### ISSUE TITLE
allow to enable/disable JS for a specific URLs or masks, like example.com/articles/*

### ISSUE THREAD

**Issue by walkky** [maximelebreton/quick-javascript-switcher#13]
Currently QJS allows to control JS only on per-domain basis, as far as I can see.

Would like to enable for eg home page example.com, disable for individual URL on the same domain, eg example.com/articles/, and still enable for pages under that url, like example.com/articles/_/_

**Comment by alon91** [comment]
Yes would love this too. Have not found a way to make this happen. What I'd love to see is for a way to change rules to --

* disable javascript on http://www.espn.com/*
* make exception and allow javascript only on http://www.espn.com/nba/scoreboard

**Comment by maximelebreton** [comment]
You have been heard in the plans for next release:
https://github.com/maximelebreton/quick-javascript-switcher/issues/35

**Comment by maximelebreton** [comment]
After some digging, unfortunately Chrome limits the pattern to domains, so we can't create specific url rules, sorry!
